File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyLowResFragment.java
package com.mercadopago.android.px.internal.features.express.slider;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.mercadopago.android.px.R;
import com.mercadopago.android.px.internal.viewmodel.drawables.AccountMoneyDrawableFragmentItem;

public class AccountMoneyLowResFragment extends AccountMoneyFragment {

    @SuppressWarnings("TypeMayBeWeakened")
    @NonNull
    public static Fragment getInstance(@NonNull final AccountMoneyDrawableFragmentItem model) {
        final AccountMoneyLowResFragment instance = new AccountMoneyLowResFragment();
        instance.storeModel(model);
        return instance;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull final LayoutInflater inflater, @Nullable final ViewGroup container,
        @Nullable final Bundle savedInstanceState) {
        return inflater.inflate(R.layout.px_fragment_account_money_low_res, container, false);
    }
}

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyLowResFragment.kt
package com.mercadopago.android.px.internal.features.express.slider

import androidx.fragment.app.Fragment
import com.mercadopago.android.px.R
import com.mercadopago.android.px.internal.viewmodel.drawables.AccountMoneyDrawableFragmentItem

internal class AccountMoneyLowResFragment : AccountMoneyFragment() {

    override fun getLayout() = R.layout.px_fragment_account_money_low_res

    companion object {
        @JvmStatic fun getInstance(model: AccountMoneyDrawableFragmentItem): Fragment {
            return AccountMoneyLowResFragment().also {
                it.storeModel(model)
            }
        }
    }
}
